src/goInstallTools.ts

```diff
@@ -443,6 +445,21 @@ export function offerToInstallTools() {
 				});
 			}
 		});
+
+		const goConfig = vscode.workspace.getConfiguration('go');
+		const usingSourceGraph = goConfig['useLanguageServer'] === true && !getAlternateLanguageServer(goConfig);
```

Comment:
should we check here also if the alternate specified is also the sourcegraph LS ?

Reply:
Done